packages/universal-components/src/RangeDatePicker/DayItemArrow.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import * as React from 'react';
+import { Animated, Easing, View, Platform } from 'react-native';
+
+import { type StylePropType, StyleSheet } from '../PlatformStyleSheet';
+import { designTokens } from '../DesignTokens';
+
+type Props = {|
+  +style?: StylePropType,
+|};
+
+type State = {|
+  scale: Animated.Value,
+|};
+
+const SHARED_ANIMATION_CONFIG = {
+  duration: designTokens.durationArrowScaleAnimation,
+  useNativeDriver: true,
+  easing: Easing.ease,
+};
+
+export default class DayItemArrow extends React.Component<Props, State> {
+  static animating: boolean;
+  state = {
+    scale: new Animated.Value(0),
+  };
+
+  componentDidMount() {
+    DayItemArrow.animating = true;
+    this.handleAnimation();
```

Comment:
If I understand it correctly animation is disabled on Android. But the sequence is still running in background on this platform. Can we disable it?

Maybe it will be handy to have something like `isAnimationEnabled` and set it `false` on Android for now. And then just work with this prop. Basically just to prevent to have `Platform.OS !== 'android'` condition in multiple places.

Reply:
Good point, fixed